### DISCUSSION TITLE
Not connecting to Bluetooth

### DISCUSSION THREAD

**Discussion by Casey130** [nefarius/DsHidMini#287] [Support]
I've followed all of the steps and the controller seems to be working wired but not connecting to Bluetooth. I've installed BthPS#Setup_x64
Not sure what to do to get this working at this point.

![Capture1](https://user-images.githubusercontent.com/129629190/229316639-6106b52f-c326-4907-a299-eac83e6aec3d.PNG)
![Capture2](https://user-images.githubusercontent.com/129629190/229316640-4dd509a8-5b8c-46d4-af6c-bad1a93444e3.PNG)

**Comment by nefarius** [comment]
Check the FAQs and help resources on vigem.org

**Reply by Casey130** [reply]
I did

**Reply by nefarius** [reply]
And? What did you do, where did you not find success? Details.

**Reply by Casey130** [reply]
Most of the FAQs are related to error codes which I am not getting.
I did try conventional pairing in the past and tried to resolve this based on what it says under _[I did everything right and it just won't connect via Bluetooth!](https://vigem.org/projects/DsHidMini/FAQ/)_ but it seems to keep pairing and showing up in the Windows settings _Other Devices_. The solution in the FAQ didn't work

**Reply by nefarius** [reply]
Make a driver trace (also on the site) and attach it here for analysis.

**Reply by Casey130** [reply]
Here it is. I plugged the controller in, connected it (working), unplugged to see if Bluetooth would connect (it didn't), then plugged in, removed device from Windows settings, unplugged to see if Bluetooth would connect (it didn't).
Edit: how do I attach this file?

**Reply by nefarius** [reply]
You simply compress it and drag and drop it into your post.

**Reply by Casey130** [reply]
[BthPS3.zip](https://github.com/ViGEm/DsHidMini/files/11130817/BthPS3.zip)

**Reply by nefarius** [reply]
It's empty.

**Reply by Kanuan** [reply]
BthPS3 doesn't appear on user's device manager

![image](https://user-images.githubusercontent.com/24910442/229323584-a331c8e2-fe0f-4ff7-92c2-c494b1c9b0d3.png)

**Reply by nefarius** [reply]
Oh haha, you're correct, no surprise it's empty with the drivers missing.

**Reply by Kanuan** [reply]
@Casey130 try reinstalling bthps3. Uninstall, reboot, reinstall, reboot, check if it appears then on the device manager

**Reply by Casey130** [reply]
Wow it works. Just needed to reinstall haha 🤦
Thanks so much both of you!
